app/models/collection.js

```diff
@@ -3,20 +3,11 @@ import DS from 'ember-data';
 export default DS.Model.extend({
     collectionName: DS.attr('string'),
     code: DS.attr('string'),
-    userGroupScopeId: DS.attr('number'),
     diciplinID: DS.attr('number'),
     timestampCreated: DS.attr('number'),
-    catalogFormatNumName: DS.attr('string'),
-    timestampModified: DS.attr('number'),
     collectionType: DS.attr('string'),
     description: DS.attr('string'),
-    estimatedSize: DS.attr('number'),
-    guid: DS.attr('string'),
-    isEmbeddedCollectingEvent: DS.attr('boolean'),
-    primaryFocus: DS.attr('string'),
-    primaryPurpose: DS.attr('string'),
-    regNumber: DS.attr('string'),
-    remarks: DS.attr('string'),
-    disciplineID: DS.attr('number'),
-    version: DS.attr('number')
-});
+    agent: DS.belongsTo('agent', {async: true}),
+    institutionType: DS.belongsTo('institution', {async: true}),
```

Comment:
I think that you'll need to update the `serializer/collection.js` to map `institutionID` to `institutionType` for this to work. See the [serializer section in the Ember gudie](https://guides.emberjs.com/v2.6.0/models/customizing-serializers/#toc_attribute-names) for more info.